Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayHelpers.h

```diff
@@ -34,17 +34,9 @@
 @class BUYCheckout;
 @class BUYShop;
 
-@interface BUYApplePayHelpers : NSObject <PKPaymentAuthorizationViewControllerDelegate>
+NS_ASSUME_NONNULL_BEGIN
 
-/**
- *  Initializes a helper to support Apple Pay
- *
- *  @param client   A configured client
- *  @param checkout The checkout which is to be completed using Apple Pay
- *
- *  @return helper object
- */
-- (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout;
+@interface BUYApplePayHelpers : NSObject <PKPaymentAuthorizationViewControllerDelegate>
```

Comment:
This class is specifically for Apple Pay, I'd prefer to make that evident from the class name